lib/board.rb

```diff
@@ -0,0 +1,73 @@
+
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+		reset!
```

Comment:
I would just call `reset!` here and delegate to that method to create the default empty board. No need to set the `@cells` instance variable here since `reset!` will do that for you.